generators/app/index.js

```diff
@@ -273,6 +273,20 @@ module.exports = JhipsterGenerator.extend({
                 local: require.resolve('../client')
             });
 
+        },
+
+        composeLanguages: function () {
```

Comment:
This should be composed in the client subgen as it needs to run for client alone generation as well